tools/msp_merge/mergeMSP.xml

```diff
@@ -0,0 +1,49 @@
+<tool id="mergemsp" name="Merge MSP Spectra" version="0.1.0" python_template_version="3.5">
+   <creator>
+        <person
+            givenName="Wudmir"
+            familyName="Rojas"
+            url="https://github.com/wverastegui"
+            identifier="0000-0001-7036-9987" />
+        <person
+            givenName="Helge"
+            familyName="Hecht"
+            url="https://github.com/hechth"
+            identifier="0000-0001-6744-996X" />
+        <organization
+            url="https://www.recetox.muni.cz/"
+            email="GalaxyToolsDevelopmentandDeployment@space.muni.cz"
+            name="RECETOX MUNI"/>
+    </creator>
+    <description>Merge MSP Spectra files</description>
+    <requirements>
+         <requirement type="package" version="0.14.0">matchms</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
```

Comment:
Detect errors could be changed to aggressive.

Reply:
Done